src/containers/Search.jsx

```diff
@@ -228,8 +247,12 @@ class Search extends Component {
 		} = this.state;
 		const result_list =
 			results && results.length !== 0
-				? results.map(
-						({ facility_type, formatted_address, geometry: { location } }, index) => {
+				? results.sort((a,b)=>a.dist-b.dist)
```

Comment:
@Jasonwei1106 Don't use JavaScript `sort` inside the `render`, Use it inside `distancesort` function.